testing/ncids-css-testing/stories/uswds-native/usa-select/src/index.scss

```diff
@@ -0,0 +1,2 @@
+@use "styles/ncids";
+@forward "usa-select";
```

Comment:
needs imports for usa-form and usa-label

uswds doesn't have separate css for each test so they are inheriting all the styles, if you check the test against https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/develop/?path=/story/components-form-inputs-select--select you'll see some missing styles